opta/commands/help.py

```diff
@@ -0,0 +1,29 @@
+# type: ignore
```

Comment:
Why are we ignoring typing in this file?

Reply:
try and see if you can reduce the scope of typing ignore and it lgtm